docs/release-notes/2.27.0.md

```diff
@@ -0,0 +1,129 @@
+# Percona Monitoring and Management 2.27.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">April 7, 2022</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of the PMM instance. This ensures that you access the latest features of the underlying components, such as VictoriaMetrics, with all the bug fixes in place. One of the critical features of VictoriaMetrics is [stream parsing mode](https://docs.victoriametrics.com/vmagent.html#stream-parsing-mode), which enhances the performance of PMM and saves memory when scraping targets expose millions of metrics.
+
+* PMM and Percona Portal Integration
+
+    - Social Login/Registration
+    - **Enhanced UX for connecting PMM server to Percona Portal**: PMM now requires access-token authorization for connecting PMM instances to Percona Platform.
+    - **More visibility for Percona customers**: Percona customers who connect their PMM instances to the Percona Platform will now be able to see their account Entitlements and Support tickets.
+    - **New Paid tier and special Advisor checks for customers**: PMM 2.27 introduces new sets of checks to cover additional database health assessments. All checks are grouped into sets of Advisors, according to the functionality and recommendations they provide. 
+
+* Advisors
+    - New presentation and tiers, including the Paid tier 
+    - Advisor checks have been extended with two new query types: getDiagnosticData and replSetGetStatus. 
+    - Documentation about developing checks 
+    - DBaaS - Simplified experience in registering EKS cluster with kubeconfig generated by eksctl.
+
+* Components upgrade
+    - Grafana: PMM 2.27.0 has now migrated to Grafana 8.3.5. This version of Grafana is loaded with a gamut of exciting features. For more information, see [What's new in Grafana v8.0](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v8-0/).
+    - VictoriaMetrics: VictoriaMetrics has been upgraded to 1.72.0.
+
+
+## New Features
```

Comment:
we again missing https://jira.percona.com/browse/PMM-8800

Reply:
Its there in Release Highlights:     - DBaaS - Simplified experience in registering EKS cluster with kubeconfig generated by eksctl.

Reply:
it is gone :(

Reply:
Even if we mention features in Highlights,  we need to keep them in the general list with features/improvements/bugs.  So highlights are the top replacement but a summary of everything bellow

Reply:
ok. But won't it be redundant?

Reply:
highlights gives more info, list of the tickets only titles